app/src/main/res/values-lt/strings.xml

```diff
@@ -1,143 +1,180 @@
 <resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="MissingTranslation">
+    <string-array name="themesEntries">
+        <item>?viesi tema (numatytoji)</item>
+        <item>Pusiau permatoma tema</item>
+        <item>Permatoma tema</item>
+        <item>Tamsi tema</item>
+        <item>Tamsi pusiau permatoma tema</item>
+        <item>Tamsi permatoma tema</item>
+    </string-array>
+    <string-array name="sortAppListEntries">
+        <item>Nuo A iki Z</item>
+        <item>Nuo Z iki A</item>
+    </string-array>
+    <string-array name="historyModeEntries">
+        <item>Paskiausias pirma</item>
+        <item>Da?niausias pirma (da?numas)</item>
```

Comment:
Not sur,e but I'm assuming Lithuanin doesn't use question mark in the middle of a word and there is an encoding issue?

Can you confirm? Thanks